src/init/wazuh/wazuh.sh

```diff
@@ -97,13 +97,22 @@ WazuhUpgrade()
         fi
     fi
 
-    # Remove existing SQLite databases
-
-    rm -f $DIRECTORY/var/db/global.db*
+    # Remove/relocate existing SQLite databases
     rm -f $DIRECTORY/var/db/.profile.db*
     rm -f $DIRECTORY/var/db/.template.db*
     rm -f $DIRECTORY/var/db/agents/*
 
+    if [ -f "$DIRECTORY/var/db/global.db" ]; then
+        cp $DIRECTORY/var/db/global.db* $DIRECTORY/queue/db/
```

Comment:
We should move `global.db` only. Discard SHM and WAL files if any.

Reply:
FIXED